projects/postgresql/Dockerfile

```diff
@@ -45,3 +47,7 @@ RUN unzip corpus_json.zip
 WORKDIR /corpus
 RUN cp -r /postgresql/src/test/regress/sql/* .
 WORKDIR /
+
+# Set up cron to clear disk from garbage every minute.
+RUN printf "* * * * * find /tmp -mmin +10 -name 'query-sydr*' -exec rm -rf {} \;" \
```

Comment:
Update docs that watch is not needed anymore

Reply:
done